client/src/assets/logo/frosh-main-logo.svg

```diff
@@ -4,7 +4,7 @@
 	 width="1217.74px" height="1218.21px" viewBox="0 0 1217.74 1218.21" style="enable-background:new 0 0 1217.74 1218.21;"
 	 xml:space="preserve">
 <style type="text/css">
-	.st0{fill:#EBEBEB;}
+	.st0{fill:#FFFFFF00;}
```

Comment:
Quick question… is it okay that the “2T2” isn’t visible anymore? 🤔

Reply:
Oh shoot my bad, I wanted to just remove the circle background
Could you submit a PR with the dino icon just without the circle background (but with 2T2)